lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -283,16 +282,16 @@ func (c *Client) start() {
 			switch m := msg.(type) {
 			case tdp.MouseMove:
 				mouseX, mouseY = m.X, m.Y
-				if err := cgoError(C.write_rdp_pointer(
+				if err := C.write_rdp_pointer(
 					c.rustClient,
 					C.CGOMousePointerEvent{
 						x:      C.uint16_t(m.X),
 						y:      C.uint16_t(m.Y),
 						button: C.PointerButtonNone,
 						wheel:  C.PointerWheelNone,
 					},
-				)); err != nil {
-					c.cfg.Log.Warningf("Failed forwarding RDP mouse pointer: %v", err)
+				); err != C.ErrCodeSuccess {
+					c.cfg.Log.Warningf("write_rdp_pointer failed on the Rust side")
```

Comment:
The new error message seems less helpful to a user, not sure they care about "on the Rust side" "on the Go side", why not leave "Failed forwarding RDP mouse pointer" here?

Reply:
[That error message has now been moved over to Rust](https://github.com/gravitational/teleport/blob/2cb663db3cda686e3e5be243ba7d94f3695e9d66/lib/srv/desktop/rdp/rdpclient/src/lib.rs#L593). Agreed that this isn't important to the user though, I will change all of these to `Debugf`'s.

Reply:
I still think there's no need to log "on the Rust side" - this is kind of irrelevant and will just confuse a user reading their logs.